qcfractal/interface/collections/dataset_view.py

```diff
@@ -3,31 +3,29 @@
 import pathlib
 import warnings
 from contextlib import contextmanager
-from typing import Any, Dict, Iterator, List, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Tuple, Union
```

Comment:
I don't think we need to import this or I am very wrong here?

Reply:
`TYPE_CHECKING` is in the namespace of the `typing` module (https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING). Or maybe you mean that there was no `if TYPE_CHECKING:` block in this file when this comment was made.

Reply:
Yup, I am just [foolish](https://github.com/samuelcolvin/pydantic/blob/master/pydantic/main.py#L10).